assertj-core/src/main/java/org/assertj/core/api/KeyStoreAssert.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2023 the original author or authors.
+ */
+package org.assertj.core.api;
+
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.cert.Certificate;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.error.KeyStoreShouldContainAlias.shouldContainAlias;
+import static org.assertj.core.error.KeyStoreShouldNotContainAlias.shouldNotContainAlias;
+
+public class KeyStoreAssert extends AbstractAssert<KeyStoreAssert, KeyStore> {
+  public KeyStoreAssert(KeyStore keyStore) {
+    super(keyStore, KeyStoreAssert.class);
+  }
+
+  public KeyStoreAssert containsEntry(String alias, Certificate[] certificateChain) {
+    isNotNull();
+
+    // FIXME implement and write tests
+
+    return this;
+  }
+
+  /**
+   * Verifies that the actual {@code KeyStore} contains the specified alias.
```

Comment:
add a code example showing a successful assertions and if that helps a failed one (probably not needed in this case but that's a general advice)

Reply:
this comment applies to all assertion methods in this file